src/battle/move/auto_multibounce_76A830.c

```diff
@@ -1,3 +1,33 @@
 #include "common.h"
 
+#ifdef NON_MATCHING
+ApiStatus func_80238000_6F10E0(ScriptInstance* script, s32 isInitialCall) {
```

Comment:
Is this the same function as func_802A1000_765C20? If so, leave a comment saying so pls.